setup.py

```diff
@@ -2,9 +2,19 @@
 
 from os.path import exists
 from setuptools import setup
+import re
+
+
+version_raw = open('chest/_version.py').read()
+version_regex = r"^__version__ = ['\"]([^'\"]*)['\"]"
+version_result = re.search(version_regex, version_raw, re.M)
+if version_result:
+    version_string = version_result.group(1)
+else:
+    raise RuntimeError("Unable to find version string in %s." % (VERSIONFILE,))
```

Comment:
Ah!  Scary.

I wonder if just storing the version as a string in a non-python file would be cleaner.